HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
I agree with @brianlheim here. Help files should be as clear as possible. This is cryptic (although I do it in my own code).